src/ragger/backend/interface.py

```diff
@@ -29,6 +29,12 @@ class RaisePolicy(Enum):
     RAISE_ALL = auto()
 
 
+class NavigationInstruction(Enum):
```

Comment:
Given navigation will probably be a thing on its own, I'd say it could deserve its own module.

Reply:
But currently it is entangled with one of the concrete method of the interface, so it would make sense _for now_ to keep it inside the `backend` package.

Reply:
@xchapron-ledger ?

Reply:
Ok, I close this discussion because the future refactoring will probably throw this out of the windows.